### DISCUSSION TITLE
施主增加桌面歌词可否？

### DISCUSSION THREAD

**Discussion by TG-Twilight** [SPlayer-Dev/SPlayer#244] [想法 Ideas]
软件已极为精妙，若能增设桌面歌词，实乃锦上添花，岂不美哉！

诚知“热词”可曲线救国，然其时常失灵，未能尽善，故此望您谅解一二。

**Comment by imsyy** [comment]
新版本已加入桌面歌词功能，待重构完成即可

![image](https://github.com/user-attachments/assets/c050a092-dab3-4010-9931-c50245bea377)
